File: 240712/짝수의 합 n번 구하기/find-the-sum-of-even-numbers-n-times.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();

        for(int i = 0; i < n; i++){
            int a = sc.nextInt();
            int b = sc.nextInt();

            System.out.println(getEvensSum(a, b));
        }
        
    }

    public static int getEvensSum(int a, int b){
        int sum = 0;
        
        for(int i = a; i <= b; i++){
            if(i % 2 == 0) sum += i;
        }

        return sum;
    }
}

File: 240712/짝수의 합 n번 구하기/find-the-sum-of-even-numbers-n-times.js
const fs = require('fs');
const arr = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(arr[0]);

const getEvensSum = (a, b) => {
    let sum = 0;
    
    for(let i = a; i <= b; i++){
        if(i % 2 == 0) sum += i;
    }

    return sum;
}

for(let i = 1; i <= n; i++){
    const [a, b] = arr[i].split(' ').map(Number);

    console.log(getEvensSum(a, b));
}